Fixing `RuntimeError: The size of tensor a (1024) must match the size…… of tensor b

Andrewlaw171 opened the pull request.

Andrewlaw171 committed:
Fixing `RuntimeError: The size of tensor a (1024) must match the size of tensor b`

timoschick commented on modeling.py:
```diff
@@ -272,7 +272,7 @@ def generate_self_debiasing(self, input_text: str, debiasing_texts: List[str], n
             min_length = min_length + input_length
         if max_length is not None:
-            max_length = max_length + input_length
+            max_length = min(1024, max_length + input_length)
```
Great, thank you! I think it would be safer to replace `1024` with `self.model._model.config.max_position_embeddings` so that this will not break for other language models with a different maximum sequence length.

Andrewlaw171 replied:
True, I hadn't thought of that. Code has been amended.

Andrewlaw171 committed:
Update modeling.py

Use self.model._model.config.max_position_embeddings instead of hardcoded value

timoschick reviewed (approved):
Looks good to me!
